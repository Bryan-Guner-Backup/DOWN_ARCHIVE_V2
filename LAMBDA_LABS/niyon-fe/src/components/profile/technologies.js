export const technology = [
  {
    value: 1,
    label: 'C ++',
    type: 'Language'
  },
  {
    value: 2,
    label: 'React',
    type: 'Framework'
  },
  {
    value: 3,
    label: 'Nodejs',
    type: 'Language'
  },
  {
    value: 4,
    label: 'Angular',
    type: 'Framework'
  },
  {
    value: 5,
    label: 'Dev-Ops',
    type: 'Soft'
  },
  {
    value: 6,
    label: 'Go!',
    type: 'Language'
  },
  {
    value: 7,
    label: 'Ruby',
    type: 'Language'
  },
  {
    value: 8,
    label: 'Graphql',
    type: 'Language'
  },
  {
    value: 9,
    label: 'JavaScript',
    type: 'Language'
  },
  {
    value: 10,
    label: 'Javascript - Front-End',
    type: 'Soft'
  },
  {
    value: 11,
    label: 'Javascript - Back-End',
    type: 'Soft'
  },
  {
    value: 12,
    label: 'Java',
    type: 'Language'
  },
  {
    value: 13,
    label: 'Kotlin',
    type: 'Language'
  },
  {
    value: 14,
    label: 'C#',
    type: 'Language'
  },
  {
    value: 15,
    label: 'Rest API',
    type: 'Soft'
  },
  {
    value: 16,
    label: 'Swift',
    type: 'Language'
  },
  {
    value: 17,
    label: 'Ember',
    type: 'Framework'
  },
  {
    value: 18,
    label: 'AWS',
    type: 'Soft'
  },
  {
    value: 19,
    label: 'SQL',
    type: 'Language'
  },
  {
    value: 20,
    label: '.NEt',
    type: 'Language'
  },
  {
    value: 21,
    label: 'TypeScript',
    type: 'Language'
  },
  {
    value: 22,
    label: 'Linux',
    type: 'Soft'
  },
  {
    value: 23,
    label: 'HTML',
    type: 'Language'
  },
  {
    value: 24,
    label: 'Docker',
    type: 'Soft'
  },
  {
    value: 25,
    label: 'PHP',
    type: 'Language'
  },
  {
    value: 26,
    label: 'Perl',
    type: 'Language'
  },
  {
    value: 27,
    label: 'Python',
    type: 'Language'
  },
  {
    value: 28,
    label: 'PowerShell',
    type: 'Soft'
  },
  {
    value: 29,
    label: 'Rust',
    type: 'Language'
  },
  {
    value: 30,
    label: 'Unix',
    type: 'Soft'
  },
  {
    value: 31,
    label: 'Visual Basic',
    type: 'Language'
  },
  {
    value: 32,
    label: 'Vim',
    type: 'Soft'
  },
  {
    value: 33,
    label: 'MATLAB',
    type: 'Language'
  },
  {
    value: 34,
    label: 'Functional Programming',
    type: 'Soft'
  },
  {
    value: 35,
    label: 'Object Oriented Programming',
    type: 'Soft'
  },
  {
    value: 36,
    label: 'Interviewing',
    type: 'Soft'
  },
  {
    value: 37,
    label: 'Communication',
    type: 'Soft'
  },
  {
    value: 38,
    label: 'Agile',
    type: 'Soft'
  },
  {
    value: 39,
    label: 'Leadership',
    type: 'Soft'
  },
  {
    value: 40,
    label: 'Integrated Development Environment',
    type: 'Soft'
  },
  {
    value: 41,
    label: 'NPM',
    type: 'Soft'
  },
  {
    value: 42,
    label: 'Express',
    type: 'Framework'
  },
  {
    value: 43,
    label: 'Rails',
    type: 'Framework'
  },
  {
    value: 44,
    label: 'Django',
    type: 'Framework'
  },
  {
    value: 45,
    label: 'Vue',
    type: 'Framework'
  },
  {
    value: 46,
    label: 'Graphql-Apollo',
    type: 'Framework'
  },
  {
    value: 47,
    label: 'Graphql-Yoga',
    type: 'Framework'
  },
  {
    value: 48,
    label: 'Spring',
    type: 'Framework'
  },
  {
    value: 49,
    label: 'Laravel Lumen',
    type: 'Framework'
  },
  {
    value: 50,
    label: 'Flask',
    type: 'Framework'
  },
  {
    value: 51,
    label: 'ASP.NET',
    type: 'Framework'
  },
  {
    value: 52,
    label: 'DRY Code',
    type: 'Soft'
  },
  {
    value: 53,
    label: 'Machine Learning',
    type: 'Soft'
  },
  {
    value: 54,
    label: 'C ++',
    type: 'Language'
  },
  {
    value: 55,
    label: 'React',
    type: 'Framework'
  },
  {
    value: 56,
    label: 'Nodejs',
    type: 'Language'
  },
  {
    value: 57,
    label: 'Angular',
    type: 'Framework'
  },
  {
    value: 58,
    label: 'Dev-Ops',
    type: 'Soft'
  },
  {
    value: 59,
    label: 'Go!',
    type: 'Language'
  },
  {
    value: 60,
    label: 'Ruby',
    type: 'Language'
  },
  {
    value: 61,
    label: 'Graphql',
    type: 'Language'
  },
  {
    value: 62,
    label: 'JavaScript',
    type: 'Language'
  },
  {
    value: 63,
    label: 'Javascript - Front-End',
    type: 'Soft'
  },
  {
    value: 64,
    label: 'Javascript - Back-End',
    type: 'Soft'
  },
  {
    value: 65,
    label: 'Java',
    type: 'Language'
  },
  {
    value: 66,
    label: 'Kotlin',
    type: 'Language'
  },
  {
    value: 67,
    label: 'C#',
    type: 'Language'
  },
  {
    value: 68,
    label: 'Rest API',
    type: 'Soft'
  },
  {
    value: 69,
    label: 'Swift',
    type: 'Language'
  },
  {
    value: 70,
    label: 'Ember',
    type: 'Framework'
  },
  {
    value: 71,
    label: 'AWS',
    type: 'Soft'
  },
  {
    value: 72,
    label: 'SQL',
    type: 'Language'
  },
  {
    value: 73,
    label: '.NEt',
    type: 'Language'
  },
  {
    value: 74,
    label: 'TypeScript',
    type: 'Language'
  },
  {
    value: 75,
    label: 'Linux',
    type: 'Soft'
  },
  {
    value: 76,
    label: 'HTML',
    type: 'Language'
  },
  {
    value: 77,
    label: 'Docker',
    type: 'Soft'
  },
  {
    value: 78,
    label: 'PHP',
    type: 'Language'
  },
  {
    value: 79,
    label: 'Perl',
    type: 'Language'
  },
  {
    value: 80,
    label: 'Python',
    type: 'Language'
  },
  {
    value: 81,
    label: 'PowerShell',
    type: 'Soft'
  },
  {
    value: 82,
    label: 'Rust',
    type: 'Language'
  },
  {
    value: 83,
    label: 'Unix',
    type: 'Soft'
  },
  {
    value: 84,
    label: 'Visual Basic',
    type: 'Language'
  },
  {
    value: 85,
    label: 'Vim',
    type: 'Soft'
  },
  {
    value: 86,
    label: 'MATLAB',
    type: 'Language'
  },
  {
    value: 87,
    label: 'Functional Programming',
    type: 'Soft'
  },
  {
    value: 88,
    label: 'Object Oriented Programming',
    type: 'Soft'
  },
  {
    value: 89,
    label: 'Interviewing',
    type: 'Soft'
  },
  {
    value: 90,
    label: 'Communication',
    type: 'Soft'
  },
  {
    value: 91,
    label: 'Agile',
    type: 'Soft'
  },
  {
    value: 92,
    label: 'Leadership',
    type: 'Soft'
  },
  {
    value: 93,
    label: 'Integrated Development Environment',
    type: 'Soft'
  },
  {
    value: 94,
    label: 'NPM',
    type: 'Soft'
  },
  {
    value: 95,
    label: 'Express',
    type: 'Framework'
  },
  {
    value: 96,
    label: 'Rails',
    type: 'Framework'
  },
  {
    value: 97,
    label: 'Django',
    type: 'Framework'
  },
  {
    value: 98,
    label: 'Vue',
    type: 'Framework'
  },
  {
    value: 99,
    label: 'Graphql-Apollo',
    type: 'Framework'
  },
  {
    value: 100,
    label: 'Graphql-Yoga',
    type: 'Framework'
  },
  {
    value: 101,
    label: 'Spring',
    type: 'Framework'
  },
  {
    value: 102,
    label: 'Laravel Lumen',
    type: 'Framework'
  },
  {
    value: 103,
    label: 'Flask',
    type: 'Framework'
  },
  {
    value: 104,
    label: 'ASP.NET',
    type: 'Framework'
  },
  {
    value: 105,
    label: 'DRY Code',
    type: 'Soft'
  },
  {
    value: 106,
    label: 'Machine Learning',
    type: 'Soft'
  },
  {
    value: 107,
    label: 'C ++',
    type: 'Language'
  },
  {
    value: 108,
    label: 'React',
    type: 'Framework'
  },
  {
    value: 109,
    label: 'Nodejs',
    type: 'Language'
  },
  {
    value: 110,
    label: 'Angular',
    type: 'Framework'
  },
  {
    value: 111,
    label: 'Dev-Ops',
    type: 'Soft'
  },
  {
    value: 112,
    label: 'Go!',
    type: 'Language'
  },
  {
    value: 113,
    label: 'Ruby',
    type: 'Language'
  },
  {
    value: 114,
    label: 'Graphql',
    type: 'Language'
  },
  {
    value: 115,
    label: 'JavaScript',
    type: 'Language'
  },
  {
    value: 116,
    label: 'Javascript - Front-End',
    type: 'Soft'
  },
  {
    value: 117,
    label: 'Javascript - Back-End',
    type: 'Soft'
  },
  {
    value: 118,
    label: 'Java',
    type: 'Language'
  },
  {
    value: 119,
    label: 'Kotlin',
    type: 'Language'
  },
  {
    value: 120,
    label: 'C#',
    type: 'Language'
  },
  {
    value: 121,
    label: 'Rest API',
    type: 'Soft'
  },
  {
    value: 122,
    label: 'Swift',
    type: 'Language'
  },
  {
    value: 123,
    label: 'Ember',
    type: 'Framework'
  },
  {
    value: 124,
    label: 'AWS',
    type: 'Soft'
  },
  {
    value: 125,
    label: 'SQL',
    type: 'Language'
  },
  {
    value: 126,
    label: '.NEt',
    type: 'Language'
  },
  {
    value: 127,
    label: 'TypeScript',
    type: 'Language'
  },
  {
    value: 128,
    label: 'Linux',
    type: 'Soft'
  },
  {
    value: 129,
    label: 'HTML',
    type: 'Language'
  },
  {
    value: 130,
    label: 'Docker',
    type: 'Soft'
  },
  {
    value: 131,
    label: 'PHP',
    type: 'Language'
  },
  {
    value: 132,
    label: 'Perl',
    type: 'Language'
  },
  {
    value: 133,
    label: 'Python',
    type: 'Language'
  },
  {
    value: 134,
    label: 'PowerShell',
    type: 'Soft'
  },
  {
    value: 135,
    label: 'Rust',
    type: 'Language'
  },
  {
    value: 136,
    label: 'Unix',
    type: 'Soft'
  },
  {
    value: 137,
    label: 'Visual Basic',
    type: 'Language'
  },
  {
    value: 138,
    label: 'Vim',
    type: 'Soft'
  },
  {
    value: 139,
    label: 'MATLAB',
    type: 'Language'
  },
  {
    value: 140,
    label: 'Functional Programming',
    type: 'Soft'
  },
  {
    value: 141,
    label: 'Object Oriented Programming',
    type: 'Soft'
  },
  {
    value: 142,
    label: 'Interviewing',
    type: 'Soft'
  },
  {
    value: 143,
    label: 'Communication',
    type: 'Soft'
  },
  {
    value: 144,
    label: 'Agile',
    type: 'Soft'
  },
  {
    value: 145,
    label: 'Leadership',
    type: 'Soft'
  },
  {
    value: 146,
    label: 'Integrated Development Environment',
    type: 'Soft'
  },
  {
    value: 147,
    label: 'NPM',
    type: 'Soft'
  },
  {
    value: 148,
    label: 'Express',
    type: 'Framework'
  },
  {
    value: 149,
    label: 'Rails',
    type: 'Framework'
  },
  {
    value: 150,
    label: 'Django',
    type: 'Framework'
  },
  {
    value: 151,
    label: 'Vue',
    type: 'Framework'
  },
  {
    value: 152,
    label: 'Graphql-Apollo',
    type: 'Framework'
  },
  {
    value: 153,
    label: 'Graphql-Yoga',
    type: 'Framework'
  },
  {
    value: 154,
    label: 'Spring',
    type: 'Framework'
  },
  {
    value: 155,
    label: 'Laravel Lumen',
    type: 'Framework'
  },
  {
    value: 156,
    label: 'Flask',
    type: 'Framework'
  },
  {
    value: 157,
    label: 'ASP.NET',
    type: 'Framework'
  },
  {
    value: 158,
    label: 'DRY Code',
    type: 'Soft'
  },
  {
    value: 159,
    label: 'Machine Learning',
    type: 'Soft'
  }
]
