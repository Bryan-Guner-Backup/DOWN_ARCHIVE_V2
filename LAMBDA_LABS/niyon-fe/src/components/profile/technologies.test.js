import React from 'react'
import toJSON from 'enzyme-to-json'
import { technology } from './technologies'

describe('Testing technologies', () => {
  it('should render technologies array correctly', () => {
    expect(technology).toEqual(expect.arrayContaining(technology))
  })

  it('should pass a snapshot test correctly', () => {
    expect(toJSON(technology)).toMatchSnapshot()
  })
})
