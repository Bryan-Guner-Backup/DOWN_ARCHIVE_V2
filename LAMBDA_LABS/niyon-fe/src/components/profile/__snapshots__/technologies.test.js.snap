// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing technologies should pass a snapshot test correctly 1`] = `undefined`;
